web/react/components/sidebar.jsx

```diff
@@ -157,6 +125,7 @@ export default class Sidebar extends React.Component {
             privateChannels,
             visibleDirectChannels,
             hiddenDirectChannelCount,
+            unreadCounts: JSON.parse(JSON.stringify(ChannelStore.getUnreadCounts())),
```

Comment:
Yes I think your right. Lets merge this and talk about it in the dev meeting today.